### PR TITLE
Add new fields (max_latency, priority) to the Flow object.

### PR DESCRIPTION
**Pull request by dpoltavets**
Closes: #2087.
Part of: #2088.

### CODE REVIEW

File: services/src/api/src/main/java/org/openkilda/northbound/dto/flows/FlowPatchDto.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+package org.openkilda.northbound.dto.flows;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+@Data
+public class FlowPatchDto {
+
+    @JsonProperty("max_latency")
```
Comment (timofei-durakov) [inline services/src/api/src/main/java/org/openkilda/northbound/dto/flows/FlowPatchDto.java]: `max-latency`

File: services/src/api/src/main/java/org/openkilda/northbound/dto/flows/FlowPatchDto.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    private Integer maxLatency;
+
+    @JsonProperty("priority")
+    private Integer priority;
+
+    @JsonCreator
+    public FlowPatchDto(@JsonProperty("max_latency") Integer maxLatency,
```
Comment (timofei-durakov) [inline services/src/api/src/main/java/org/openkilda/northbound/dto/flows/FlowPatchDto.java]: the same

File: services/src/api/src/main/java/org/openkilda/northbound/dto/flows/FlowPatchDto.java (hunk 1 above)
Comment (dpoltavets) [reply services/src/api/src/main/java/org/openkilda/northbound/dto/flows/FlowPatchDto.java]: Thank you! Fixed.

File: services/src/api/src/main/java/org/openkilda/northbound/dto/flows/FlowPatchDto.java (hunk 2 above)
Comment (dpoltavets) [reply services/src/api/src/main/java/org/openkilda/northbound/dto/flows/FlowPatchDto.java]: Fixed.

**Pull request merged**